docs/guides/basics/data-types.md

```diff
@@ -39,28 +39,54 @@ For a formula the value type is specified in the `resultType` property, and for
 [View all types][ValueType]{ .md-button }
 
 
-### Arrays
+### Strings
 
-Lists of data can be returned using the [`Array`][Array] value type. You must also specify the value type of the items in the array, using the `items` property.
+Text values can be returned as [JavaScript strings][mdn_string] using the [`String`][String] value type.
 
 ```ts
 pack.addFormula({
   // ...
-  resultType: coda.ValueType.Array,
-  items: { type: coda.ValueType.String },
+  resultType: coda.ValueType.String,
   execute: async function ([], context) {
-    return ["This", "is", "a", "string", "array"];
+    return "Hello World!";
   },
+});
+```
+
+
+### Numbers
+
+Number values can be returned as [JavaScript numbers][mdn_number] using the [`Number`][Number] value type.
+
+```ts
+pack.addFormula({
   // ...
+  resultType: coda.ValueType.Number,
+  execute: async function ([], context) {
+    return 42;
+  },
 });
 ```
 
-The resulting array is represented as a [`List`][formula_list] in the Coda formula language.
+
+### Booleans
+
+True or false values can be returned as [JavaScript booleans][mdn_boolean] using the [`Boolean`][Boolean] value type.
+
+```ts
+pack.addFormula({
+  // ...
+  resultType: coda.ValueType.Boolean,
+  execute: async function ([], context) {
+    return true;
+  },
+});
+```
 
 
 ### Objects
 
-Structured data can be returned using the [`Object`][Object] value type. These objects must conform to an existing schema, as specified in the `schema` property. See the [Schemas guide][schemas] for more information about defining and using schemas.
+Structured data can be returned as [JavaScript objects][mdn_object] using the [`Object`][Object] value type. These objects must conform to an existing schema, as specified in the `schema` property. See the [Schemas guide][schemas] for more information about defining and using schemas.
```

Comment:
"existing schema" is maybe a bit confusing--is this trying to say a "known schema" or "explicitly-defined schema", i.e. "you can't just return an object you also have to describe the schema"? Would maybe be a bit more specific.

Reply:
Changed to "defined schema", which I think captures it OK.